source/common/conn_pool/conn_pool_base.cc

```diff
@@ -132,8 +132,8 @@ ConnPoolImplBase::tryCreateNewConnection(float global_preconnect_ratio) {
     return ConnectionResult::ShouldNotConnect;
   }
 
-  const bool can_create_connection =
-      host_->cluster().resourceManager(priority_).connections().canCreate();
```

Comment:
Is this called in other filters (redis? Not sure) that might also need to be updated and/or should we specify this only applies to certain protocols also?

Reply:
yeah I think everything is switched over to the new pool so shares this code.